pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/RuntimeASTTransformer.groovy

```diff
@@ -67,7 +67,7 @@ import static org.objectweb.asm.Opcodes.ACC_PUBLIC
  * {@link ExecutionModelAction} to the run.
  */
 @SuppressFBWarnings(value="SE_NO_SERIALVERSIONID")
-class RuntimeASTTransformer {
+public class RuntimeASTTransformer {
```

Comment:
guessing you do not actually want this to be public (except for test code?) so you can change it at will?
```suggestion
@Restricted(NoExternalUse.class) // for test use only
public class RuntimeASTTransformer {
```

Reply:
@jtnord 
Actually, no.  I considered that, but Groovy that is loaded and run in pipeline has difficulty with attributes.  Making this public is not great, but not horrible.